connectors/src/connectors/microsoft/index.ts

```diff
@@ -25,8 +25,8 @@ import {
   getFolders,
   getSites,
   getTeams,
-  microsoftInternalIdFromNodeData,
-  microsoftNodeDataFromInternalId,
+  internalId,
```

Comment:
IMHO, `internalId` seems not self explanatory

Reply:
Ok for createInternalId?

Reply:
went with `internalIdFromTypeAndPath` so the symetry with `typeAndPathFromInternalId` is clear